modules/cert-manager-proxy-support.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-proxy-support_{context}"]
+= TODO: Title
+
+TODO: intro
+
+////
+If a cluster wide egress proxy is configured on the OpenShift cluster, OLM automatically update all the operators' deployments with HTTP_PROXY, HTTPS_PROXY, NO_PROXY environment variables.
+Those variables are then propagated down to the cert-manager (operand) controllers by the cert manager operator.
+Trusted Certificate Authority
+Running operator
+
+Follow the instructions below to let Cert Manager Operator trust a custom Certificate Authority (CA). The operator's OLM subscription has to be already created.
+////
+
+.Prerequisites
+
+* TODO
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a config map in the `cert-manager` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create configmap trusted-ca -n cert-manager
+----
+
+. Inject the CA bundle that is trusted by {product-title} into the config map by running the following command:
++
+[source,terminal]
+----
+$ oc label cm trusted-ca config.openshift.io/inject-trusted-cabundle=true -n cert-manager
+----
+
+. Update the deployment for the {cert-manager-operator} to use the config map by running the following command:
++
+[source,terminal]
+----
+$ oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
+----
++
+////
+This failed for me:
+[ahoffer@ahoffer march2] $ oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
+Error from server (NotFound): subscriptions.operators.coreos.com "cert-manager-operator" not found
+////
+
+. TODO: Wait for rollout. Command to check? Or say wait <x> minutes?
+
+.Verification
+
+* Verify that the CA bundle was added by running the following command:
++
+[source,terminal]
+----
+$ oc get deployment cert-manager -n cert-manager -o=jsonpath={.spec.template.spec.'containers[0].volumeMounts'} | jq
+----
++
+// TODO: Update to not use jq
++
+.Example output
+[source,terminal]
+----
+[
+  {
+    "mountPath": "/etc/pki/tls/certs/cert-manager-tls-ca-bundle.crt",
+    "name": "trusted-ca",
+    "subPath": "ca-bundle.crt"
+  }
+]
+----
+
+* Verify (TODO: What is this command verifying compared to the other one?):
```

Comment:
@thejasn @swghosh What is this verifying compared to the above command?

Reply:
One verifies wether the ca bundle has been mounted as a volume (above) and the below command verifies the source of the ca bundle i.e. from a configmap named `trusted-ca`.